ui/apps/platform/src/Containers/Vulnerabilities/components/FilterAutocomplete.tsx

```diff
@@ -41,6 +43,7 @@ function FilterAutocompleteSelect({
     searchFilter,
     setSearchFilter,
     searchOptions,
+    useRegexMatch = true,
```

Comment:
1. Because our eyes are trained to read `useRegexMatch` as a hook, what do you think about `is` prefix?
2. What are pro and con to make the prop required?

Reply:
1. Oh wow good call - I wonder if there is a linter rule that can catch naming with a `use` prefix when not in the context of hooks?
2. I can make this required - this was a leftover from when I defaulted it to `false`, but I decided to call this out explicitly in the Exception Management pages for clarity.